k8s/base/config/actions-server-application.yml

```diff
@@ -1,9 +1,28 @@
 server:
   port: 80
 
+spring:
+  jpa:
+    database: POSTGRESQL
+    show-sql: false
+    hibernate:
+      ddl-auto: validate
+      naming:
+        physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
+        implicit-strategy: org.hibernate.boot.model.naming.ImplicitNamingStrategyJpaCompliantImpl
+    properties:
+      dialect: org.hibernate.dialect.PostgreSQLDialect
+      hibernate.format_sql: false
+  datasource:
```

Comment:
i think we should externalize that data source and use  a specific database.properties file instead.
so  add a database.properties file in actions-server

Reply:
Done